picard/tagger.py

```diff
@@ -386,12 +386,19 @@ def get_album_pane_tracks(self):
             for track in album.iterfiles():
                 yield track
 
+    def get_all_file_objects(self):
+        return self.unclustered_files.files + [x for x in self.get_album_pane_tracks()] + [x for x in self.clusters.iterfiles()]
```

Comment:
I think that's equivalent to:

```suggestion
        return self.unclustered_files.files + list(self.get_album_pane_tracks()) + list(self.clusters.iterfiles())
```